packages/tools/devtools/devtools-view/src/components/ContainerHistoryLog.tsx

```diff
@@ -98,7 +98,7 @@ export function ContainerHistoryLog(props: ContainerHistoryLogProps): React.Reac
 			}
 			case "disconnected": {
 				// orange
-				return tokens.colorPaletteDarkOrangeBorderActive;
+				return tokens.colorPaletteDarkOrangeBorder1;
```

Comment:
Can we add a unit test to ensure the contrast ratio doesn't regress?

Reply:
I don't think this is worth adding regression tests for. If Fluent UI's color palette changes, we can update our tokens to point to a different color.